.github/workflows/release.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-          token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

Comment:
Should we delete `PERSONAL_ACCESS_TOKEN` from the secrets?

Reply:
Let's keep it for now. If bot publisher doesn't work we'll return it back.